misc/dbt-materialize/dbt/include/materialize/macros/deploy/deploy_init.sql

```diff
@@ -177,13 +179,141 @@
 
             {% set create_cluster %}
                 CREATE CLUSTER {{ deploy_cluster }} (
-                    SIZE = '{{ size }}',
+                    SIZE = {{ dbt.string_literal(size) }},
                     REPLICATION FACTOR = {{ replication_factor }}
                 );
             {% endset %}
             {{ run_query(create_cluster) }}
             {{ set_cluster_ci_tag() }}
+            {{ internal_copy_cluster_grants(schema, deploy_schema) }}
         {% endif %}
     {% endif %}
 {% endfor %}
 {% endmacro %}
+
+{% macro internal_copy_schema_default_privs(from, to) %}
+
+{% set find_default_privs %}
+SELECT
+  'ALTER DEFAULT PRIVILEGES ' ||
+  CASE
+    WHEN object_owner = 'PUBLIC' THEN 'FOR ALL ROLES '
+    ELSE 'FOR ROLE ' || object_owner
+  END ||
+  'IN SCHEMA {{ to }} '         ||
+  'GRANT '  || privilege_type   || ' ' ||
+  'ON '     || object_type      || 's ' ||
+  'TO '     || grantee
+FROM mz_internal.mz_show_default_privileges
+WHERE database = current_database() AND schema = {{ dbt.string_literal(from) }}
+    AND object_owner <> 'none' AND grantee <> 'none'
+{% endset %}
+
+{% set alter_defaults = run_query(find_default_privs) %}
+{% if execute %}
+    {% for alter in alter_defaults.rows %}
+        {{ run_query(alter) }}
+    {% endfor %}
+{% endif %}
+{% endmacro %}
+
+{% macro internal_copy_schema_grants(from, to) %}
+{% set find_revokes %}
+WITH schema_privledge AS (
```

Comment:
s/privledge/privilege/ (here and a few more times below)

Reply:
Ugh. Sorry.